schemas/session-replay/common/view-end-record-schema.json

```diff
@@ -6,7 +6,7 @@
   "description": "Schema of a Record which signifies that view lifecycle ended.",
   "allOf": [
     {
-      "$ref": "_common-record-schema.json"
+      "$ref": "_webview-supported-common-record-schema.json"
```

Comment:
hmmmm why do we want to alter all the other records by inheriting this `webview_supported` schema ?

Reply:
the way I see it this will modify also the mobile records format ?

Reply:
On the first hand, we want to alter only the records that could have a `generatedByWebViewId` field: browser records. The Mobile FS and Incremental S records would never have such field so I think it's better to even give the possibility in the schemas.

On the other, for the shared records like the Metadata that is both generated from the Native Mobile Replay recorder and the Browser replay recorder, we have to enable this optinal field.

Indeed, you don't want to apply the Metarecord of the webview on the main Native replay but want to apply it inside the webview encapsulation: we therefore need the webview id.

Another approach would be to split the Metarecord between the BrowserMetarecord (that could have the generatedByWebViewId) and MobileMetarecord (that could not have it) but it would require introducing duplication and new record types.

Reply:
From my perspective, the concept of Meta record is not Mobile or Browser related. It's Replay related. On top of that, for the replay player it has the exact same goal for mobile and browser so I don't think we should split it.

For specific records like Mobile FS and Mobile incrementals, you do not have the optional field as you still inherit from the `_common-record-schema.json` so it sounds all good isn't it ?